requirements/common.txt

```diff
@@ -2,12 +2,10 @@ psutil
 cython
 wheel
 requests
-pillow==9.3.0
+pillow
 captum
 packaging
 numpy; sys_platform != 'win32'
 numpy==1.19.3; sys_platform == 'win32' #see https://tinyurl.com/y3dm3h86
-nvgpu; sys_platform != 'win32'
-nvgpu==0.8.0; sys_platform == 'win32'
```

Comment:
I'm not sure this will work, the existing `metric_collector.py` imports nvgpu at the top, might need to make the import conditional and make sure the rest of the metric infra doesn't break cc @namannandan

Reply:
I see it being importing in `system_metrics.py` and it is handled correctly